src/generate-trigger-properties/generate.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-relative-packages */
+import fs from "fs"
+import triggerConfig from "../../output-data/data/excluded-trigger-config.json"
+import type { ExcludedTriggerConfig, TriggerCode } from "../../output-data/types/types"
+
+const typedTriggerConfig = <ExcludedTriggerConfig>triggerConfig
+
+const generateForceConfig = (
+  force: string,
+  triggers: TriggerCode[],
+  test: boolean = false
+): string => {
+  const includeRule: string[] = [`trigger.rule.${force}=include`]
+  if (force === "01" && test) {
```

Comment:
Not sure I quite understand what's going on here - are we saying that when test is true, the trigger properties for force 01 should be empty, but all others should be the same as normal?

Reply:
Correct - at the moment, the tests all rely on a specific `test.triggers.properties` file which is configured so that force 01 has all triggers